javascripts/chooseplayer.js

```diff
@@ -9,16 +11,22 @@ var FoodFight = (function(choose){
     FoodFight.setUpGame(currentHero, currentEnemy);
   });
 
+  heroBtn.click(function() {
+    selected = heroBtn.filter(":checked").val();
+    console.log(selected);
+  });
+
   choose.setHero = function(){
-    var hero = FoodFight.Larry;
+    var name = userName.val()
+    var hero = new FoodFight[selected](name);
     return hero;
-    //find the thing that is chosen and set it to hero. 
+    //find the thing that is chosen and set it to hero.
   };
 
   choose.generateEnemy= function(){
     var enemy = FoodFight.player2;
```

Comment:
I'm not sure about this one yet. We'll really be able to figure it out once everything is merged